src/middleware/router.js

```diff
@@ -501,9 +497,19 @@ async function sendHttpRequest (ctx, route, options) {
     responseProgress: function (chunk, counter, size) {
       logger.info(`Write response CHUNK # ${counter} [ Total size ${size}]`)
     },
-    finishResponse: function () {
+    finishResponse: function (response, size) {
```

Comment:
It doesnt seem like we make use of these parameters. Are they needed?

Reply:
These parameters are passed in the call. They may or may not be used in the event handler. Some of the handlers have no code; the parameters are just included to document the what information is available in the call...

Reply:
Ok cool :+1